pkgs/development/python-modules/pyinfra/default.nix

```diff
@@ -0,0 +1,59 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pythonOlder
+, gevent
```

Comment:
Put the attrs from gevent in alphabetical ordering, one per line.

Reply:
Addressed with c85bebe